processing/src/main/java/io/druid/query/BaseQuery.java

```diff
@@ -62,6 +62,11 @@
     return parseBoolean(query, "finalize", defaultValue);
   }
 
+  public static <T> int getContextPopulateUncoveredIntervals(Query<T> query, int defaultValue)
```

Comment:
sure, i can rename the key to `uncoveredIntervalsLimit` if that is more appropriate.